run_kraken.py

```diff
@@ -37,9 +39,39 @@ def main(cfg):
     logging.info("Starting kraken")
 
     # Parse and read the config
-    if os.path.isfile(cfg):
-        with open(cfg, "r") as f:
-            config = yaml.full_load(f)
+    # Initializing flag variable as 0 to test config is a file or not
+    flag = 0
```

Comment:
flag might not be required, we can just exit after raising an exception.

Reply:
Actually flag was set to test config is a file or not. If it is, all yaml file is loaded in the config variable and it goes on scenarios part otherwise it goes into the else part and exit from there.

Reply:
do not think we need flag this if the code is exiting on an exception